lms/djangoapps/ccx/migrations/0003_add_master_course_staff_in_ccx.py

```diff
@@ -5,8 +5,8 @@
 from django.db import migrations
 from django.http import Http404
 
-from lms.djangoapps.courseware.courses import get_course_by_id
 from lms.djangoapps.ccx.utils import add_master_course_staff_to_ccx, remove_master_course_staff_from_ccx
+from openedx.core.lib.courses import get_course_by_id
```

Comment:
Remind me - changing imports in a migration is a no-op/won't cause issues?

Reply:
It should be fine. My understanding is that Django just compares the file/migration name against its internal db state table (`django_migrations`?) and it ignores the contents, eg, they aren't comparing file hashes, just file names.

Reply:
Neat, I guess I've always wondered about that